### PR TITLE
ticket 4 added a list of reviews

### PR DESCRIPTION
**Pull request by ElenaCKay**

### CODE REVIEW

File: src/components/reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { fetchReviews } from "../api";
+
+const Reviews = () => {
+    const [reviewsData, setReviews] = useState([]);
+
+    useEffect(() => {
+        fetchReviews().then((data) => {
+            setReviews(data);
+        });
+    });
```
Comment (B7Bprog) [inline src/components/reviews.jsx]: Currently your useEffect() function is missing the dependency array, causing an infinite re-render and an infinite fetch loop. Please include it. ([]).

File: src/components/reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { fetchReviews } from "../api";
+
+const Reviews = () => {
+    const [reviewsData, setReviews] = useState([]);
+
```
Comment (B7Bprog) [inline src/components/reviews.jsx:7]: Another state needs to be added for letting the user know if the reviews are still loading.

File: src/components/reviews.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    useEffect(() => {
+        fetchReviews().then((data) => {
+            setReviews(data);
+        });
+    });
+
+    return (
+        <div>
```
Comment (B7Bprog) [inline src/components/reviews.jsx]: Please use conditional rendering here so the user will be notified if the reviews are still loading.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Elena, nice work so far, please see my comments and amend your code, then send us another PR with it. 
(You can also consider utilising the review_ing_url property so you can display an image for all reviews.)

**Review by robert112233 — APPROVED** [review]
You've implemented the suggested feedback well done, merge 👍

**Pull request merged**